src/mach/fat.rs

```diff
@@ -58,8 +60,7 @@ impl FatHeader {
 }
 
 #[repr(C)]
-#[derive(Clone, Copy, Default)]
-#[cfg_attr(feature = "std", derive(Pread, Pwrite, SizeWith))]
+#[derive(Clone, Copy, Default, Pread, Pwrite, SizeWith)]
```

Comment:
If my understanding is correct these are unconditionally derived because in order for the module to be compiled it must have mach and alloc feature, yes? In which case we are safe to derive them. 

If so I almost prefer the explicit cfg feature since although it’s redundant it gives reader a clue what’s going on. Or in case of refactors it’s a second fail safe of sorts.

Reply:
Yes, `mach` depends on `alloc`, and `alloc` depends on `scroll/derive`. I kind of agree with you, except that there's a ton of these in `load_command.rs` and I was lazy.  Additionally, for the ones in `load_command.rs` we would also need `cfg_attr(feature = "std", derive(IOread, IOwrite))` (I don't actually understand why they don't give errors currently...).

Reply:
Yea be lazy, I forgot there’s like 40 of them ;)